addon/utils/on-transition-end.js

```diff
@@ -30,23 +30,34 @@ const transitionEndEventName = findTransitionEventName();
  * @method onTransitionEnd
  * @param {Element} element
  * @param {Function} callback
- * @param {String} transitionProperty
- * @param {Boolean} once
+ * @param {Object} options
  */
-export default function onTransitionEnd(element, callback, transitionProperty = 'all', once = false) {
-	const fn = (e) => {
-		const { propertyName, type } = e;
+export default function onTransitionEnd(element, callback, options = {}) {
+	const { transitionProperty = 'all', once = false, onlyTarget = false } = options;
+
+	function fn(e) {
```

Comment:
What if this function is renamed as `onTransitionCallback` or similar to ease the debugging??